ci/recipe/meta.yaml

```diff
@@ -36,6 +36,7 @@ requirements:
     - qiime2 {{ qiime2_epoch }}.*
     - q2templates {{ qiime2_epoch }}.*
     - q2-types {{ qiime2_epoch }}.*
+    - lxml
```

Comment:
Where is this dependency being used? Or is this just q2-bloatware...

Reply:
Shouldn't need to be here anymore now that the actual file reading was moved to the framework